broker/setup.py

```diff
@@ -29,7 +30,6 @@
     ],
     install_requires=[
         'netfilter>=0.6.2',
-        'six>=1.10.0',
```

Comment:
Why can this be removed?

Reply:
This is a Python3 to Python2 compatibility layer. Unless we want to support Python 2 as well we don't need six.